### ISSUE TITLE
[Feature Request]: 

### ISSUE THREAD

**Issue by TarandeepKang** [jasp-stats/jasp-issues#2155]
### Description

 Spearman-Brown coefficient

### Purpose

New split-half reliability coefficient

### Use-case

Especially useful in contexts where scales have only two items

### Is your feature request related to a problem?

The currently implemented methods seem suboptimal for calculating reliability coefficients where scales have only two items

### Is your feature request related to a JASP module?

Factor

### Describe the solution you would like

Implement the Spearman-Brown coefficient and possibly other split-half methods

### Describe alternatives that you have considered

Use SPSS or R

### Additional context

The Spearman-Brown coefficient has been shown to be superior to other methods for reliability calculation of scales with only two items, and is currently available in Pronk's splithalfr package 

Brown, W. (1910). Some Experimental Results in the Correlation of Mental Abilities1. British Journal of Psychology, 1904-1920, 3(3), 296–322. https://doi.org/10.1111/j.2044-8295.1910.tb00207.x
Eisinga, R., Grotenhuis, M. te, & Pelzer, B. (2013). The reliability of a two-item scale: Pearson, Cronbach, or Spearman-Brown? International Journal of Public Health, 58(4), 637–642. https://doi.org/10.1007/s00038-012-0416-3
Pronk, T., Molenaar, D., Wiers, R. W., & Murre, J. (2022). Methods to split cognitive task data for estimating split-half reliability: A comprehensive review and systematic assessment. Psychonomic Bulletin & Review, 29(1), 44–54. https://doi.org/10.3758/s13423-021-01948-3
Spearman, C. (1910). Correlation Calculated from Faulty Data. British Journal of Psychology, 1904-1920, 3(3), 271–295. https://doi.org/10.1111/j.2044-8295.1910.tb00206.x
Warrens, M. J. (2016). A comparison of reliability coefficients for psychometric tests that consist of two parts. Advances in Data Analysis and Classification, 10(1), 71–84. https://doi.org/10.1007/s11634-015-0198-6

**Comment by juliuspfadt** [comment]
@TarandeepKang thanks for the request. This seems quite easy to implement although I think it could also easily be calculated by hand, or maybe I am missing something, but if `r` is the correlation between the two items, the spearman brown coefficient is given by 
```r
sb = 2*r / (1 + r)
```
I would this when I implement some other coefficients.

**Comment by TarandeepKang** [comment]
Hi Julius,

That is exactly correct. I'm not suggesting that this is any way massively urgent, it would just be nice to have at some point.

**Comment by wolfvanpaemel** [comment]
hi!, while you're at it, maybe add guttman's split half coefficient as well (aka his lambda_4)?

**Comment by juliuspfadt** [comment]
Hi @wolfvanpaemel, I started adding lambda_4 in the underlying R-package but then stopped because it performs so very poorly. The same could be said about the glb which is also still in there. I would rather also remove the glb than add another coefficient that largely overestimates the reliability.

**Comment by wolfvanpaemel** [comment]
good point. i was suggesting it to increase compatibility with spss, not because it's so useful

**Comment by wolfvanpaemel** [comment]
to follow up, i am not sure how the underlying code works, but *if* you would add the spearman brown coefficient, the lambda 4 coefficient would logically have to be included as well, because of the freedom that users have in choosing standardized vs unstandardized coefficients (with spearman brown being the standardized version of lambda 4)
